examples/3dtiles_hierarchy.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Itowns - 3d-tiles hierarchy example</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/loading_screen.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div class="help">
+            <p><b>Feature Information:</b></p>
+            <div id="featureInfo"></div>
+        </div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/loading_screen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // Note: The following positions and camera parameters have been
+            // obtained manually using view.controls.getCameraCoordinate() ;
+            // view.controls.getCameraTargetPosition() ;
+            // view.controls.getCameraOrientation()
+            var positionOnGlobe = {
+                longitude: -75.61349109370566,
+                latitude: 40.04425929988183,
+                altitude: 45.52006505522877
+            };
+
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            var view = new itowns.GlobeView(viewerDiv, positionOnGlobe);
+
+            var cameraLookAt = new itowns.THREE.Vector3(1215018.0820894851,
```

Comment:
Could you `view.controls.lookAtCoordinate` to set camera position .
You could use `view.controls.getLookAtCoordinate ( )`  to get the parameters to use.

Reply:
I'm sorry I don't understand your comment, could you be more specific please ?

Reply:
Could you use `view.controls.lookAtCoordinate({coord: new itowns.Coordinates('EPSG:4326', longitude, latitude), range: 500, tilt: 10, heading}, false)` to place camera.

Reply:
Ok, thanks, done

Reply:
I had not seen the variable `positionOnGlobe`, you could simply write:
```js
var positionOnGlobe = {
    longitude: -75.61127,
    latitude: 40.04190,
    altitude: 0,
};
view.controls.lookAtCoordinate({ range: 400, tilt: 10, heading: -145 }, false);
```

Reply:
Done, thanks it's cleaner this way